1-js/05-data-types/01-primitives-methods/article.md

```diff
@@ -1,12 +1,12 @@
 # Methods of primitives
 
-JavaScript allows to work with primitives (strings, numbers etc) as if they were objects.
+JavaScript allows us to work with primitives (strings, numbers etc) as if they were objects.
 
-They also provide methods to call and such. We are going to study them soon, but first let's see how it works, because, of course, primitives are not objects (and here we plan to make it even more clear).
+They also provide methods to call and such. We will study those soon, but first we'll see how it works, because, of course, primitives are not objects (and here we will make it even more clear).
 
 [cut]
 
-Let's formulate the key distinction between primitives and objects.
+Let's look at the key distinction between primitives and objects.
```

Comment:
Why 'formulate' is bad here?

Reply:
Formulate indicates creating mathematical operation, whereas this is comparing the differences, 'looking at' the differences makes more sense.

Reply:
ok